frontend/src/components/docs/DocsPage.tsx

```diff
@@ -239,115 +237,117 @@ const DocsPage: React.FC = () => {
   // This HTML is rendered directly from our own docs file, so this is safe
   const localClasses = useLocalStyles();
   return (
-    <Paper className={localClasses.docs}>
```

Comment:
I just changed the Paper to a Card/CardContent, most of the changes are just indentation